src/test/java/testsuite/regression/StatementRegressionTest.java

```diff
@@ -10252,13 +10252,14 @@ public void testBug74932() throws Exception {
         int currPrepCount = this.rs.getInt(2);
         for (int i = 0; i < 10; i++) {
             testBug74932ExecuteStmts(testConn, sql1, sql2);
-
             this.rs = testConn.createStatement().executeQuery("SHOW STATUS LIKE 'Prepared_stmt_count'");
             assertTrue(this.rs.next());
             assertEquals(2, this.rs.getInt(2) - currPrepCount);
         }
         testConn.close();
 
+        Thread.sleep(5000);
```

Comment:
I think we should discuss these sleep statements offline

Reply:
Instead of adding delays can we add detailed logs that would help us root cause the flakiness? @sergiyv-bitquill what do you think?

Reply:
As discussed: need comment to explain why this sleep is needed.